packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1887,46 +1873,36 @@ export const audiusBackend = ({
     }
   }
 
-  /**
-   * Returns whether the current user is subscribed to userId.
-   */
-  // TODO(C-2719)
-  async function getUserSubscribed(userId: ID) {
```

Comment:
was this just not used? awesome

Reply:
Ha yes. I think there are several more items in ABE that were converted to SDK long ago and never got removed.